Formula/rpm.rb

```diff
@@ -28,6 +27,8 @@ def install
 
     # only rpm should go into HOMEBREW_CELLAR, not rpms built
     inreplace ["macros.in", "platform.in"], "@prefix@", HOMEBREW_PREFIX
+    # ensure that pkg-config binary is found for dep generators
+    inreplace "scripts/pkgconfigdeps.sh", "/usr/bin/pkg-config", Formula["pkg-config"].bin/"pkg-config"
```

Comment:
`configure` should autodetect `pkg-config` if it's in `PATH`.
It is a mistake to hard-code pkg-config here, which should be reported upstream if it breaks stuff.
(But it was present in the previous version too)

Reply:
hurray for 10 years of a hard-coded path https://github.com/rpm-software-management/rpm/commit/6c4b0fc9e44ea3b9449e171404c1b2037d15d01e

Reply:
Yup. It is something I am gong to look to address, but this worked for now.